run.py

```diff
@@ -0,0 +1,77 @@
+import autoUniCalendar
+import os
+import uuid
+import utils
+import re
+
+
+from flask import Flask, render_template, request, send_file
+from flask_talisman import Talisman
+app = Flask(__name__)
+Talisman(app, content_security_policy=None)
+
+defaultFilename = autoUniCalendar.filename
+debug = os.environ.get('FLASK_ENV') == 'development'
+
+@app.route('/', methods = ['GET'])
+def index():
+    return serve()
+
+@app.route('/', methods = ['POST'])
+def form_post():
+    if debug: print(f"[DEBUG] POST data received from React: {request.form}")
```

Comment:
React is not implemented in this version.

Reply:
just a generic debug message, it returns anything received from the `POST` request. will update anyway.

Reply:
solved in https://github.com/miermontoto/epiCalendar/commit/88fe237b4ae2619554d26a34e316bda7b8bf41c4